_dashboards/drag-drop-wizard.md

```diff
@@ -13,39 +13,52 @@ You can use the drag and drop visualization wizard in OpenSearch Dashboards to c
 
 The drag and drop visualization wizard:
 
-* Gives you an immediate view of your data without the need to preselect the visualization output 
-* Gives you the flexibility to change visualization types and index patterns on the fly
-* Gives you the ability to easily navigate between multiple screens 
+* Gives you an immediate view of your data without the need to preselect the visualization output. 
+* Gives you the flexibility to change visualization types and index patterns on the fly.
```

Comment:
Can we use something other than the idiomatic "on the fly"? "quickly"? "in real time"?